promgen/templates/promgen/alert_row.html

```diff
@@ -0,0 +1,21 @@
+{% verbatim %}
+<td>
+    <a :href="alert.generatorURL" v-text="alert.startsAt"></a>
+</td>
+<td>
+    <dl class="dl-horizontal">
+        <dt>Labels</dt>
+        <dd>
+            <ul class="list-inline">
+                <li v-for="(value, label, index) in alert.labels">
+                    <a @click.prevent="silenceAppendLabel" class="label label-warning" v-bind:data-label="label" v-bind:data-value="value">{{label}}:{{value}}</a>
```

Comment:
Since v-on is omitted, it is better to omit v-bind.

Reply:
`v-on` has a shortcut as `@` so `@click.prevent` is the same as `v-on:click.prevent`. The `v-bind:data-label` are so I can read those values from the click event

Reply:
I know that v-bind also has a shortcut.

https://vuejs.org/v2/guide/syntax.html#v-bind-Shorthand

Reply:
Ah, yes I should probably be more consistent with using short forms or not. Do you have a preference for short vs long forms ?

Reply:
I like short form better.
However, if the long form is used in the project, we will use that.
Short form was used in this PR, so I wrote this comment.